### PR TITLE
DEV-50961: Add global default PR template

### PR DESCRIPTION
**Pull request by mattpbryant**
### Description of your changes

1. Add the Collibra PR template here so that it is applied by default to all repositories. This will only apply to repos that do not have their own PR template. Doing this will also allow quicker iterations on the template since updates will need to be made in fewer places. 
2. Add PEC as codeowners of this repo.

### JIRA reference

[DEV-50961](https://engineering-collibra.atlassian.net/browse/DEV-50961)

---

### Impact Analysis

This will impact all repos that do not already have a PR template in place. 

---

### Type of change

- [ ] Feature: _Non-breaking change which adds functionality_
- [ ] Fix: _Non-breaking change which fixes an issue_
- [ ] Documentation: _Changes only related to updating, adding or removing docs_
- [ ] Style: _Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)_
- [ ] Refactor: _A code change that neither fixes a bug nor adds a feature_
- [ ] Performance: _A code change that improves performance_
- [ ] Test: _Adding missing or correcting existing tests_
- [x] Chore: _Changes to the build process or auxiliary tools and libraries such as documentation generation_
- [ ] Breaking change: _Fix or feature that would cause existing functionality to not work as expected_
- [ ] This change requires a documentation update

---

#### Checklist

- [x] This PR is Ready for Review
- [x] I have squashed my commits into logical, [atomic commits]
- [x] I have performed a self-review of my code
- [x] My code follows the style/formatting guidelines of this project
- [x] My changes generate no new warnings
- [ ] I have made corresponding changes to the documentation (_if applicable_)

[atomic commits]: https://engineering-collibra.atlassian.net/wiki/spaces/CFT/pages/15588884570/Atomic+commits

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @collibra/principal-engineering-committee
```
Comment (vini-nair) [inline CODEOWNERS:1]: **question** : This team in gitHub is Niels, Steven and Mieszko. Not sure if this list is appropriate.

Comment (mattpbryant) [reply CODEOWNERS:1]: Happy to adjust but not sure what the right group is here?

Comment (vini-nair) [reply CODEOWNERS:1]: We should create a Jira for CICD team to add the rest of the PEC to this group. https://engineering-collibra.atlassian.net/wiki/spaces/CPL/pages/12041133905/CICD+Support

Comment (mattpbryant) [reply CODEOWNERS:1]: Sure, can do 👍 I believe I also made everyone in the group maintainers when I created it too, so the PEC can add other members themselves in future.

Comment (rathpc) [reply CODEOWNERS:1]: updated members to correctly reflect current members

**Comment by danutzcodrescu** [comment]
closing in favour of #2

File: CODEOWNERS (hunk 1 above)
Comment (NielsDoucet) [reply CODEOWNERS:1]: > **question** : This team in gitHub is Niels, Steven and Mieszko. Not sure if this list is appropriate.

interesting 🤔

**Pull request closed without merging**